OWNERS_ALIASES

```diff
@@ -1,20 +1,116 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 aliases:
-  approvers:
-      - davidvossel
+
+  #
```

Comment:
supernit - A quick statement here about the differences between SMEs and SIGs here might be useful.

Reply:
+1